### ISSUE TITLE
Lightbulb Service not userInteractive

### ISSUE THREAD

**Issue by florian-muc** [devbobo/homebridge-lifx-lan#10]
Recently discovered that some apps don't show the LIFx lightbulb in the app and did some testing.

When inquiring on iOS, the LIFx Lightbulb service reports being not userInteractive. When looking at HMService.h, this is correct behavior.

This means it's a bug and the plugin should return YES for userInteractive.

/*!
- @brief Indicates if this service supports user interaction or not.
  *
- @discussion Applications should use this property to filter out services that the users
-             should not directly interact with, e.g. HMServiceTypeAccessoryInformation.
  */
  @property(readonly, getter=isUserInteractive, nonatomic) BOOL userInteractive NS_AVAILABLE_IOS(9_0);

**Comment by devbobo** [comment]
@florian-mac which apps and what iOS version ?

**Comment by devbobo** [comment]
@florian-muc I spoke with @khaost about this problem, he said accessories need to provide a service name, as the presence of this determines if it's user interactive or not.

My current release (0.1.4) should address [this](https://github.com/devbobo/homebridge-lifx-lan/blob/master/index.js#L176), but the devices will need to be freshly instanciated.

To do this easily..

1 Make the following code changes [here](https://github.com/devbobo/homebridge-lifx-lan/blob/master/index.js#L135)...

``` javascript
    this.api.on('didFinishLaunching', function() {
        this.api.unregisterPlatformAccessories("homebridge-lifx-lan", "LifxLan", this.accessories);
        this.accessories = {};

        /*
        Client.init({
            debug:                  this.config.debug || false,
            broadcast:              this.config.broadcast || '255.255.255.255',
            lightOfflineTolerance:  this.config.lightOfflineTolerance || 5,
            messageHandlerTimeout:  this.config.messageHandlerTimeout || 45000,
            resendMaxTimes:         this.config.resendMaxTimes || 4,
            resendPacketDelay:      this.config.resendPacketDelay || 150
        });
        */
    }.bind(this));
```

2 Restart homebridge twice (ensures that devices aren't reloaded from the device cache).
3 Remove code changes above.
4 Restart homebridge

**Comment by devbobo** [comment]
@florian-muc I've just release version `0.1.5` which should automatically add the Service name if one doesn't exist, when Homebridge is restarted.

**Comment by florian-muc** [comment]
@devbobo Thanks so much for the quick reaction and fix :) Already updated to 0.1.5, all is perfect now. Thanks again!
